<launch>
  <arg name="i" default="9"/>
  <arg name="d" default="80"/>

  <node pkg="map_server" type="map_server" name="map_server" args="$(find map_transform)/maps/maze6-3.png 0.1" respawn="false" > 
    <param name="frame_id" value="/map" />
  </node>

  <node pkg="map_transform" type="visibility" name="visibility" output="screen">    
    <param name="defl" value="$(arg d)" />
    <param name="infl" value="$(arg i)" />
    <param name="debug" value="true" />
    <param name="show" value="true" />
    <param name="x" value="50" />
    <param name="y" value="50" />
    <param name="ground_truth" value="false" />
    <remap from="/visibility/map" to="/obstacle_map" />
  </node>

  <node pkg="map_transform" type="obs-gen" name="obstacle_generator" output="screen"/>

  <param name="complete_analysis" value="True" />
  <node pkg="map_transform" type="map-planner" name="planner" output="screen">       
    <remap from="/map" to="/visibility/v_map" />
  </node>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="gui"/>
</launch>
